MediaBrowser.Common/Net/NetworkConfiguration.cs

```diff
@@ -173,4 +173,14 @@ public string BaseUrl
     /// Gets or sets a value indicating whether <seealso cref="RemoteIPFilter"/> contains a blacklist or a whitelist. Default is a whitelist.
     /// </summary>
     public bool IsRemoteIPFilterBlacklist { get; set; }
+
+    /// <summary>
+    /// Gets or sets a value indicating whether public user listing is enabled.
+    /// </summary>
+    public bool PublicUserListing { get; set; } = true;
```

Comment:
I prefer to have a single enum instead of two booleans for this option. `Disabled` `Local only` and `Always` describes the behavior easier than two booleans.